src/js/promise.rei

```diff
@@ -15,7 +15,7 @@ type promise(+'a) = rejectable('a, never);  /* Internal; use Promise.t. */
 type t(+'a) = promise('a);
 
 
-
+type exn;
```

Comment:
This looks suspicious. Isn't type `exn` predefined? What version of BuckleScript are you using?

Unless something has changed upstream, I don't see why this should now become a problem, after having worked all this time, and it is especially concerning that it's a problem with `exn`.

Reply:
I was using bs-platform@3.3

Reply:
i just checked it again here https://github.com/idkjs/promise. No issues. Will revert if it happens again.